container/root/run.d/99-xdebug.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [[ ($CFG_APP_DEBUG = 1 || $CFG_APP_DEBUG = '1' || $CFG_APP_DEBUG = 'true') && ($CFG_XDEBUG_ENABLE = 1 || $CFG_XDEBUG_ENABLE = '1' || $CFG_XDEBUG_ENABLE = 'true') ]]
+then
+  echo '[debug] Enabling XDebug extension'
+  sed -i 's/^;zend_extension/zend_extension/' $CONF_PHPMODS/xdebug.ini
```

Comment:
Alpine does not have the `phpenmod` command, so we mount the file with the extension "disabled" via semicolon by default.  In all builds, we need to remove this semi colon when turning XDebug on.

In Alpine's case, that is enough.  For the others where `phpenmod` is available, that command must still be run to perform the symlinking.